app/src/org/commcare/android/util/FileUtil.java

```diff
@@ -395,4 +396,20 @@ public static String getDumpDirectory(Context c){
                 return null;
             }
         }
+
+    public static Properties loadProperties(Context c) throws IOException {

```

Comment:
Looks like some people use the NDK to obscure their keys:

http://stackoverflow.com/questions/27243896/c-c-code-in-order-to-store-private-keys-in-android-app

I think its a matter of how much we want to put into this